frontends/parsers/v1/v1parser.ypp

```diff
@@ -268,10 +268,12 @@ header_type_declaration: HEADER_TYPE name "{" header_dec_body "}"
                                 new IR::Annotations(*$4.annotations), *$4.fields),
             new IR::Type_Header(@1+@5, IR::ID(@2, $2),
                                 new IR::Annotations(*$4.annotations), *$4.fields))); }
+    | HEADER_TYPE name "{" header_dec_body error END
```

Comment:
Sometimes you set `$$`, but sometimes you don't...

Reply:
The non-terminals that have a type need to set `$$` to something for every action.  The non-terminals that don't have a type can't set `$$`...